### DISCUSSION TITLE
How to obtain the parameter AUC0-24h when the simulated medication time is two weeks?

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1482] [Questions & Problems]
Hello, I'm modeling with PK-SIM recently, and I need to use the parameter AUC0-24h for calculation. Do AUC_tD1-tD2 represent AUC0-24h in the pharmacokinetic parameters of population simulation?

**Comment by Yuri05** [comment]
duplicate of https://github.com/Open-Systems-Pharmacology/Forum/discussions/1479
closing
